plugins/board-resources/src/components/EditCard.svelte

```diff
@@ -35,15 +61,100 @@
 </script>
 
 {#if object !== undefined}
-  <Grid column={1} rowGap={1.5}>
-    <EditBox
-      label={board.string.CardName}
-      bind:value={object.title}
-      icon={board.icon.Card}
-      placeholder={board.string.CardPlaceholder}
-      maxWidth="39rem"
-      focus
-      on:change={() => change('title', object.title)}
-    />
-  </Grid>
+  <Scroller>
+    <div class="flex-col-stretch h-full w-165 p-6">
+      <!-- TODO cover -->
+      <div class="close-button">
+        <Button icon={IconClose} kind="transparent" size="large" on:click={() => dispatch('close')} />
+      </div>
+      <div class="flex-row-streach">
+        <div class="w-9">
```

Comment:
Why don't use Grid?

Reply:
I've recommended to not use Grid, since css is more powerful.